app/javascript/src/lib/OWLanguageLinter.ts

```diff
@@ -244,9 +244,11 @@ function checkMixins(content: string): void {
 }
 
 function checkTranslations(content: string): void {
-  const regex = /(@translate)[\s\n]*\(/g
+  const regex = /(@translate)(\.static)?\s*\(/g
```

Comment:
Match group 1 is used for knowing the length of the whole `@translate`/`@translate.static` directive (without the parenthesis or any whitespace after it). This is used in the diagnostics `to` field, for example.

```suggestion
  const regex = /(@translate(\.static)?)\s*\(/g
```